mailpoet/tests/unit/WPCOM/DotcomHelperFunctionsTest.php

```diff
@@ -0,0 +1,45 @@
+<?php declare(strict_types = 1);
+
+namespace unit\WPCOM;
```

Comment:
It looks like the majority of the unit tests use a namespace starting with `MailPoet`. Does the `unit` namespace have any special meaning? I honestly haven't done much in the unit tests directory and I suspect it doesn't matter but just wanted to double check.

Reply:
I didn't notice. Changed.